packages/tracing/src/integrations/node/express.ts

```diff
@@ -259,6 +259,24 @@ function instrumentRouter(appOrRouter: ExpressRouter): void {
   }
 
   const router = isApp ? appOrRouter._router : appOrRouter;
+
+  if (!router) {
+    /*
+    If we end up here, this means likely that this integration is used with Express 3 or Express 5.
+    For now, we don't support these version (3 is very old and 5 is still in beta). To support Express 5,
+    we'd need to make more changes to the routing instrumentation because the router is no longer part of
+    the Express core package but maintained in its own package. The new router has different function
+    signatures and works slightly differently, demanding more changes than just taking the router from
+    `app.router` instead of `app._router`.
+    @see https://github.com/pillarjs/router
```

Comment:
Damn this kinda sucks. Maybe another reason to have an express package?

Reply:
Yup, probably. Let's chat about this at Daily/planning today 👍